debian/postrm

```diff
@@ -16,19 +16,16 @@ remove_logs(){
     rm -f /var/log/ubuntu-advantage.log*
 }
 
-remove_postinst_created_files(){
-    # config files created in postinst, need explicit handling on purge
+remove_gpg_files(){
     rm -f /etc/apt/trusted.gpg.d/ubuntu-advantage-esm-infra-trusty.gpg
```

Comment:
I think we need to make this match any /etc/apt/trusted.gpg.d/ubuntu-advantage-*.gpg  file because other gpg files will be created here for each apt-based UA service we deliver (and enable after initial attach)